### PR TITLE
LRSD-6605 [Learn] Navigation tabs are not responsive

### PR DESCRIPTION
**Pull request by GabrielFernandoS**

### CODE REVIEW

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css [hunk 1]
```diff
@@ -1550,6 +1550,14 @@ pre[class*='language-'] {
 		width: 2rem;
 	}
 
+	.page-tabs{
+		@media (max-width: 770px) {
```
Comment (rtienpal) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css:1554]: As I showed you the reason, please change this to 768px

**Comment by rtienpal** [comment]
Besides the max-width change, please fix title and commit message

**Comment by GabrielFernandoS** [comment]
ci:test:sf

**Comment by rtienpal** [comment]
ci:test:sf

**Pull request closed without merging**